omigami/models/__init__.py

```diff
@@ -58,4 +59,7 @@ def _make_estimator_from_string(estimator: str, random_state: RandomState) -> Es
     if estimator == ESTIMATORS.XGBC:
         xgbc = XGBClassifier()
         return ScikitLearnEstimator(xgbc, random_state)
+    if estimator == ESTIMATORS.PLSR:
+        plsr = PLSRegressor()
+        return ScikitLearnEstimator(plsr, random_state)
```

Comment:
is it necessary to wrap it twice? with the `PLSRegressor` and later with the `SKlearnEstimator`

Reply:
in other words, would you need much code from the SKLearnEstimator to be put in the PLS wrappers to make them usable without wrapping them?

Reply:
I think so because we would have to at least overwrite the predict method and the feature_importances property of the SklearnEstimator. So the benefit is not entirely clear to me. Also, I don't see this class really as a wrapper but more as an extension of the sklearn one in which we basically patch the predict method and add an attribute (the same we added to the classification extension). In this way, there is also a nice symmetry between the classificator and the regressor